### PR TITLE
Added permuted MNIST and CIFAR10 (RGB+gray)

### PR DESCRIPTION
**Pull request by GioZerini**
Added psmnist experiment and cifar10 experiment with two different versions: grayscale and RGB.

### CODE REVIEW

File: acds/benchmarks/cifar10.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            validation. Defaults to 10.
+        grayscale (bool): If True, convert the images to grayscale. Default to False.
+    """
+    if grayscale:
+        transform = transforms.Compose([
+            transforms.Grayscale(num_output_channels=1),
+            transforms.ToTensor()
+        ])
+    else:
+        transform = transforms.ToTensor()
```
Comment (AndreaCossu) [inline acds/benchmarks/cifar10.py]: Can you also add the normalization for RGB CIFAR? This is to be added after conversion to tensor (using Compose).

transforms.Normalize(mean=(0.4914, 0.4822, 0.4465), std=(0.247, 0.243, 0.261))

File: acds/benchmarks/mnist.py [hunk 2]
```diff
@@ -17,13 +17,26 @@ def get_mnist_data(
         bs_test (int): Batch size for the validation and test dataloaders.
         valid_perc (int): Percentage of the train dataset to use for
             validation. Defaults to 10.
+        permuted (bool): Whether to apply permutation to images.
     """
+    permutation = torch.randperm(28 * 28) if permuted else None 
+    def permute_image(image): 
+        if permutation is not None:
+            image = image.view(-1)[permutation].view(1, 28, 28)
```
Comment (AndreaCossu) [inline acds/benchmarks/mnist.py:25]: This looks ok, manually check that the output is correct to be extra sure

**Review by AndreaCossu — COMMENTED** [review]
Everything looks correct, I made a few minor comments.

File: acds/benchmarks/cifar10.py (hunk 1 above)
Comment (GioZerini) [reply acds/benchmarks/cifar10.py]: Added this line of code

File: acds/benchmarks/mnist.py (hunk 2 above)
Comment (GioZerini) [reply acds/benchmarks/mnist.py:25]: Checked

**Review by AndreaCossu — APPROVED** [review]

**Pull request merged**